build.sbt

```diff
@@ -290,12 +290,15 @@ lazy val ocs2 = project
 lazy val ephemeris = project
   .in(file("modules/ephemeris"))
   .enablePlugins(AutomateHeaderPlugin)
-  .dependsOn(coreJVM, db, sql)
+  .dependsOn(coreJVM % "compile->compile;test->test", db, sql)
   .settings(commonSettings)
   .settings(
     libraryDependencies ++= Seq(
-      "org.typelevel" %% "cats-testkit" % catsVersion % "test"
-    )
+      "co.fs2"        %% "fs2-io"              % fs2Version     % "test",
+      "org.http4s"    %% "http4s-blaze-client" % http4sVersion,
+      "org.typelevel" %% "cats-testkit"        % catsVersion    % "test"
+    ),
+    testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-l", "gem.test.Tags.RequiresNetwork") // by default, ignore network tests
```

Comment:
This will skip tests tagged with `RequiresNetwork`.  I couldn't figure out how to explicitly include these tests on the command line though, since you can't seem to override this value with `-n` as I had expected.

Reply:
The one that asks for 90K elements is kind of slow.  It didn't seem kosher to me to fail the build if you don't have network access or if there is an issue with Horizons.  I guess I think about these issues more than most people since I live in the country and only have spotty access.  If you guys prefer, I could turn it around and leave them on by default and then I could explicitly turn them off when I run it at home.

Reply:
No it's fine with me to leave them out by default. Can we pass something to sbt to make them run in CI?

Reply:
I'll work on it some more.  I was defeated on a first pass at it.